spec.md

```diff
@@ -210,10 +210,40 @@ how platforms might expose these values to their users.
 | metadata | JSON object | An opaque object of metadata for a service plan. Controller treats this as a blob. Note that there are [conventions](profile.md#service-metadata) in existing brokers and controllers for fields that aid in the display of catalog data. |
 | free | boolean | When false, service instances of this plan have a cost. The default is true. |
 | bindable | boolean | Specifies whether service instances of the service plan can be bound to applications. This field is OPTIONAL. If specified, this takes precedence over the `bindable` attribute of the service. If not specified, the default is derived from the service. |
+| [schemas](#schema-object) | object | Schema definitions for service instances and bindings for the plan. |
 
 
 \* Fields with an asterisk are REQUIRED.
 
+##### Schema Object
+
+| Response field | Type | Description |
+| --- | --- | --- |
+| [service_instance](#service-instances-object) | object | The schema definitions for creating and updating a service instance. |
+| [service_binding](#service-bindings-object) | object | The schema definition for creating a service binding. Used only if the service plan is bindable. |
+
+
+##### Service Instances Object
+
+| Response field | Type | Description |
+| --- | --- | --- |
+| [create](#input-parameters-object) | object | The schema definition for creating a service instance. |
+| [update](#input-parameters-object) | object | The schema definition for updating a service instance. |
+
+
+##### Service Bindings Object
+
+| Response field | Type | Description |
+| --- | --- | --- |
+| [create](#input-parameters-object) | object | The schema definition for creating a service binding. |
+
+
+##### Input Parameters Object
+
+| Response field | Type | Description |
+| --- | --- | --- |
+| parameters | JSON schema object | The schema definition for the input parameters. Schema definitions MUST be valid [JSON Schema draft v4](http://json-schema.org/). It is RECOMMENDED that broker authors explicitly mention the JSON schema draft 4 version, as this will prevent breaking changes when in the future this spec starts supporting future versions of JSON schema with low readability impact. Each input parameter is expressed as a property within a JSON object. Note that UIs MAY not abide by the ordering of input parameters within the schema definition. A service supporting no input parameters SHOULD provide the following schema: `{ "$schema": "http://json-schema.org/draft-04/schema#", "type": "object", "additionalProperties": false }`. |
+
```

Comment:
draft 6 is the current version that is being pushed at http://json-schema.org and we might want to in particular figure out what it would mean if I used "$schema": "http://json-schema.org/draft-06/schema#" instead of v4.

We might also consider starting our support at v6 instead of v4